### PR TITLE
Improve readme file

### LINKED ISSUE #21: Improve Readme
I think our README.md is very poor, so I think that we could follow README.md from another project like [tangerine.nvim](https://github.com/udayvir-singh/tangerine.nvim)

### PR DESCRIPTION
**Pull request by vinibispo**
Close #21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,16 +28,60 @@ use {
...
+
+---
+
+# Commands
+
+## Run
+
+### :RubyRun
+
+Runs given file (current or another) in a floating window
```
Comment (cuducos) [inline README.md]: Why is this different from [the Vim doc](https://github.com/vinibispo/ruby.nvim/blob/debfa50ac82bff053c803f030c40f064e26f2a60/doc/help.txt#L54)?

File: README.md [hunk 2]
```diff
@@ -2,15 +2,24 @@
...
 
+# Topics
+
+:large_blue_diamond: [Motivation](#motivation)
+:large_blue_diamond: [Installation](#installation)
+:large_blue_diamond: [Setup](#setup)
+:large_blue_diamond: [Commands](#commands)
+
+---
+
```
Comment (cuducos) [inline README.md]: Total detail… but I don't think these horizontal dividers help a lot (same for similar lines). The following heading top margin and bottom border create a nice visual organization already IMHO.

File: README.md [hunk 3]
```diff
@@ -19,16 +28,60 @@ use {
...
+ruby_nvim.setup({
+  test_cmd = "ruby" -- command that :RubyTest is gonna run,
+  test_args = {} -- args that :RubyTest is gonna run
+})
+```
+
+---
+
+# Commands
```
Comment (cuducos) [inline README.md:61]: 100% a matter of style and personal preference, but I think this structure involving three levels of headings (_Commands, Run, :RubyRun_, for example) makes it difficult to read and to get an overall of what the plugin offers.

I tend to prefer a concise table, as in `yaml.nvim`:

> | Command | Description |
> |:--|:--|
> | `:YAMLView` | Shows the full path and value of the current key/value pair |
> | `:YAMLYank [register]` | Yanks the full path and value of the current key/value pair. The default register is the unnamed one (`"`) |
> | `:YAMLYankKey [register]` | Yanks the full path of the key for the current key/value pair. The default register is the unnamed one (`"`) |
> | `:YAMLYankValue [regster]` | Yanks the value of the current key/value pair. The default register is the unnamed one (`"`) |
> | `:YAMLQuickfix` | Generates a quickfix with key/value pairs |
> | `:YAMLTelescope` | Full path key/value fuzzy finder via [Telescope](https://github.com/nvim-telescope/telescope.nvim) **if installed** |

File: README.md (hunk 1 above)
Comment (vinibispo) [reply README.md]: Because we have written wrong in vimdoc

File: README.md (hunk 2 above)
Comment (vinibispo) [reply README.md]: I'll fix it

File: README.md (hunk 3 above)
Comment (vinibispo) [reply README.md:61]: I'll fix it

File: README.md (hunk 1 above)
Comment (cuducos) [reply README.md]: Got it! Since we expect an optional argument, I think we can have `:RubyRun [file]` instead of just `:RubyRun` ; )

Comment (vinibispo) [reply README.md]: Ok, I'mma change it

File: README.md [hunk 4]
```diff
@@ -19,16 +30,40 @@ use {
...
+```
+
+
+# Commands
+
 
-That is totally based on [go.nvim](https://github.com/npxbr/go.nvim).
+| Command | Description |
+|:--|:--|
+| `:RubyRun [file]` | Runs given file (current or another) in a floating window |
```
Comment (cuducos) [inline README.md]: Wording suggestion:

```suggestion
| `:RubyRun [file]` | Runs `file` in a floating window (default: current file). |
```

File: README.md [hunk 5]
```diff
@@ -19,16 +30,40 @@ use {
...
+| `:RubyAlternate` | Alternate between a Ruby file and its test file. |
```
Comment (cuducos) [inline README.md]: Wording suggestion:

```suggestion
| `:RubyAlternate` | Alternate between a Ruby file and its test file, and vice-versa. |
```

File: README.md [hunk 6]
```diff
@@ -19,16 +30,40 @@ use {
...
+| `:RubyTest[!]` | Runs the test for the current file. If the file is not a test file, it tries to to find the (alternate) test file. If run with `!` and it is a test files, runs only the test under the cursor. |
+| `:RubyBrowseGem` | Searches gem that is on cursor line in https://rubygems.org |
```
Comment (cuducos) [inline README.md]: Wording suggestion:

```suggestion
| `:RubyBrowseGem` | Searches gem under the cursor line in [rubygems.org](https://rubygems.org). |
```

**Review by cuducos — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (vinibispo) [reply README.md]: I'mma fix it

File: README.md (hunk 5 above)
Comment (vinibispo) [reply README.md]: I'mma fix it

**Pull request merged**